lock-impl/src/test/java/com/palantir/lock/logger/LockServiceStateLoggerTest.java

```diff
@@ -113,14 +113,14 @@ public void testLockState() {
                 .map(request -> loggedState.getLockDescriptorMapping().get(request.getLockDescriptor()))
                 .collect(Collectors.toSet());
         Set<LockDescriptor> outstandingDescriptors = getOutstandingDescriptors();
-        assertThat(outstandingDescriptors).isEqualTo(loggedOutstandingRequests);
+        assertThat(outstandingDescriptors).containsExactlyInAnyOrderElementsOf(loggedOutstandingRequests);
```

Comment:
I guess this is to get better failure messages, even though Set equality does feel more intuitive/readable to me